test/test_optim.py

```diff
@@ -1291,6 +1298,55 @@ def closure():
             res2 = optim_neg_inf.step(closure)
             self.assertEqual(type(res1), type(res2))
 
+    @onlyCPU
+    def test_grad_scaling_autocast_fused_optimizers(self):
+        optimizer_kwargs = ({"fused": True, "amsgrad": False}, {"fused": True, "amsgrad": True})
+        separate_unscale = (False, True)
+        for kwargs, _separate_unscale in list(product(optimizer_kwargs, separate_unscale)):
+            self._grad_scaling_autocast_fused_optimizers(
+                optimizer_ctor=torch.optim.Adam, optimizer_kwargs=kwargs, separate_unscale=_separate_unscale)
+
+    def _grad_scaling_autocast_fused_optimizers(self, optimizer_ctor, optimizer_kwargs, separate_unscale):
```

Comment:
Could this be shared by other tests? If not, make it a local function of `test_grad_scaling_autocast_fused_optimizers`?

Reply:
Unified them after fused SGD/AdamW kernel is also enabled on CPU